containers/podman-desktop
# feat: adds images under manifest

feat: adds images under manifest

### What does this PR do?

* Adds "children" images under the main manifest

### Screenshot / video of UI



![Screenshot 2024-06-10 at 11 43 55 AM](https://github.com/containers/podman-desktop/assets/6422176/52ec60ff-34bd-435d-b383-5e67c62f927e)

https://github.com/containers/podman-desktop/assets/6422176/0077cb19-7cf2-4c0f-bce8-ee020b243907



### What issues does this PR fix or reference?



Closes https://github.com/containers/podman-desktop/issues/7181

### How to test this PR?




1. Create a manifest
2. No longer see the <none> images and it will instead be under the
   manifest image.

Signed-off-by: Charlie Drage <charlie@charliedrage.com>
